### PR TITLE
Update volume.md

### PR DESCRIPTION
**Pull request by varunturlapati**

### CODE REVIEW

File: control/volume.md [hunk 1]
```diff
@@ -475,6 +513,32 @@ Volume	:  970758537931791410
 			Node 	 :  10.99.117.133
 			Node 	 :  10.99.117.137
 ```
+For a volume created with a --group option the inspect output will reflect the flag as shown below:
+```
+sudo /opt/pwx/bin/pxctl volume inspect volFinGrp
```
Comment (balapx) [inline control/volume.md]: volFinGrp needs to be replaced by vol1

Comment (varunturlapati) [inline control/volume.md]: Done!

**Comment by varunturlapati** [comment]
@michaelferranti I added a new section under the CLIs but it is not showing on the left navigation bar. For ex, under CLI reference there is a section "credentials". In the repo, I added a file under control/credentials.md
Not sure what I am missing. Could you take a look please?

**Comment by ferrantim** [comment]
@varunturlapati you just need to add the page to `home_sidebar.yml` That is the file that generates the sidebar, it is not autogenerated from the folder structure.  Just add the title and url to that file and check it locally to make sire it looks how you want.  Let me know if you have any trouble.

**Comment by varunturlapati** [comment]
@michaelferranti - Awesome! Thanks a lot!

**Pull request merged**